index.Rmd

```diff
@@ -99,7 +99,6 @@ income_map <- tract_income %>%
   st_transform("+proj=longlat +datum=WGS84") %>%
   leaflet() %>%
   setView(lng = -78.878738, lat = 42.880230, zoom = 7) %>%
-  setMaxBounds(lng1 = -78.878738, lat1 = 45.027500, lng2 = -78.878738, lat2 = 40.440624) %>%
```

Comment:
The setMaxBounds call is not allowing the maps to be moved around. Whenever the user tries to move them around, it snaps back to the center you have defined. This is okay when zoomed far out, but cuts off data when you want to zoom in on specific tracts.